Vagrantfile

```diff
@@ -0,0 +1,128 @@
+  # using a specific IP.
+  # config.vm.network "private_network", ip: "192.168.33.10"
+
+  # Create a public network, which generally matched to bridged network.
+  # Bridged networks make the machine appear as another physical device on
+  # your network.
+  # config.vm.network "public_network"
+
+  # Share an additional folder to the guest VM. The first argument is
+  # the path on the host to the actual folder. The second argument is
+  # the path on the guest to mount the folder. And the optional third
+  # argument is a set of non-required options.
+  # config.vm.synced_folder "../data", "/vagrant_data"
+
+  # Provider-specific configuration so you can fine-tune various
+  # backing providers for Vagrant. These expose provider-specific options.
+  # Example for VirtualBox:
+  #
+  # config.vm.provider "virtualbox" do |vb|
+  #   # Display the VirtualBox GUI when booting the machine
+  #   vb.gui = true
+  #
+  #   # Customize the amount of memory on the VM:
+  #   vb.memory = "1024"
+  # end
+  #
+  # View the documentation for the provider you are using for more
+  # information on available options.
+
+  # Enable provisioning with a shell script. Additional provisioners such as
+  # Ansible, Chef, Docker, Puppet and Salt are also available. Please see the
+  # documentation for more information about their specific syntax and use.
+  # config.vm.provision "shell", inline: <<-SHELL
+  #   apt-get update
+  #   apt-get install -y apache2
+  # SHELL
+
+#   config.vm.provision "shell", inline: <<-SHELL
+#   	apt-get --assume-yes  install jq
+#
+#   	export DATA_FOLDER=/opt/chimera/data
+#
+#   	echo "export DATA_FOLDER=/opt/chimera/data" >> .bash_profile
+#   	echo "export PATH=$PATH:/opt/chimera/bin" >> .bash_profile
+#
+#   	mkdir -p /opt/chimera/bin
+#   	mkdir -p /opt/chimera/data
+#           cp /vagrant/provision/cliapp /opt/chimera/bin
+#           cp /vagrant/provision/webapp /opt/chimera/bin
+#           cp /vagrant/provision/all_hour_feed.sh .
+#   	chmod +x all_hour_feed.sh
+#
+#   	mkdir -p /var/log/chimera
+#   	chmod 777 /var/log/chimera
+#           chmod 777 /opt/chimera/data
+#
+#            /opt/chimera/bin/webapp &
+#     SHELL
+
+    config.vm.provision "shell", privileged: false, inline: <<-SHELL
+            sudo apt-get update
+
+          # Install pyenv prerequisites
+          sudo apt-get install -y build-essential libssl-dev zlib1g-dev libbz2-dev \
+          libreadline-dev libsqlite3-dev wget curl llvm libncurses5-dev libncursesw5-dev \
+          xz-utils tk-dev libffi-dev liblzma-dev python-openssl git
+
+          # Install pyenv
+
+          git clone https://github.com/pyenv/pyenv.git $HOME/.pyenv || true
+
+          echo 'export PYENV_ROOT=$HOME/.pyenv' >> ~/.bash_profile
+          echo 'export PATH="$PYENV_ROOT/bin:$PATH"' >> ~/.bash_profile
+          echo 'eval "$(pyenv init -)"' >> ~/.bash_profile
+          source ~/.bash_profile
```

Comment:
You should actually be updating .profile rather than .bash_profile.

Bash will load .bash_profile if it exists or otherwise load the .profile. Until you started using .bash_profile, it didn't exist, but .profile did.

So now your bash shells are missing out on some functionality that already existed in .profile.